cmd/talosctl/cmd/talos/image.go

```diff
@@ -443,6 +448,14 @@ var imageCacheServeCmd = &cobra.Command{
 			return fmt.Errorf("failed to create development logger: %w", err)
 		}
 
+		if err = generateMirrorsConfigPatch(
+			imageCacheServeCmdFlags.address,
+			imageCacheServeCmdFlags.mirrors,
+			imageCacheServeCmdFlags.tlsCertFile != "" && imageCacheServeCmdFlags.tlsKeyFile != "",
+		); err != nil {
+			development.Error("failed to generate Talos config patch for registry mirrors", zap.Error(err))
```

Comment:
This is not a fatal error, we don't want to crash image-cache server here, so just log it.